bin/pulsar

```diff
@@ -177,17 +177,23 @@ if [ ! -f "${PY_INSTANCE_FILE}" ]; then
     PY_INSTANCE_FILE=${BUILT_PY_INSTANCE_FILE}
 fi
 
-# find pulsar sql presto distribution location
-check_presto_libraries() {
-    if [ ! -d "${PRESTO_HOME}" ]; then
-
-        BUILT_PRESTO_HOME="${SQL_HOME}/presto-distribution/target/pulsar-presto-distribution"
-        if [ ! -d "${BUILT_PRESTO_HOME}" ]; then
-            echo "\nCouldn't find presto distribution.";
-            echo "Make sure you've run 'mvn package'\n";
-            exit 1;
+# find pulsar sql trino distribution location
+check_trino_libraries() {
+    if [ ! -d "${TRINO_HOME}" ]; then
+        # TODO: As PIP-200 accepted, this compatibility is not promised. Refactor when we drop this b/w compatibility.
+        FALLBACK_PRESTO_HOME="${PULSAR_HOME}/lib/presto"
```

Comment:
This should be internal dependency and users should not and cannot manage it.

Pulsar SQL tightly couples with the bundled trino distro.

Will revert the "compatibility" part.

Reply:
Concretely, <= 2.11 works with prestosql 334 or 333 under lib/presto, >= 2.12 works with trino 353 with breaking changes.

By no means one uses 2.12 wants to "fallback" to the previous lib folder.

Reply:
Revert at 54fb61e330d2bc963234727e53b3440c651906ae